packages/a11y-base/test/focus-utils.test.js

```diff
@@ -44,6 +44,11 @@ describe('focus-utils', () => {
     it('should return false for visible elements', () => {
       expect(isElementHidden(element)).to.be.false;
     });
+
+    it('should return false when using fixed positioning', () => {
+      element.style.position = 'fixed';
+      expect(isElementHidden(element)).to.be.false;
```

Comment:
Also wanted to check the child's visibility here, but in case of these nested divs, the child's offset parent is actually the parent div. In case of tabs being used within the app layouts drawer there must be some additional detail that causes the offset parent to become null.